mod_reforged/hooks/states/world_state.nut

```diff
@@ -1,4 +1,10 @@
 ::Reforged.HooksMod.hook("scripts/states/world_state", function(q) {
+	q.startNewCampaign = @(__original) function()
+	{
+		__original();
+		this.setAutoPause(false);	// Vanilla Fix: Fix scenarios without intro event locking up the game
```

Comment:
For vanilla fixes please label the comment as `VanillaFix` without space so we can always search for that term.

Reply:
We are using "Vanilla Fix" with a space more than we use it without one

Reply:
Then all of those should be changed to without space.

Reply:
I've changed all the occurrences in the code to `VanillaFix`.